docker-compose.yml

```diff
@@ -31,8 +31,9 @@ services:
       - ollama:/root/.ollama
     ports:
       - "11434:11434"
-    command: >
-      sh -c "ollama start llama3 --port 11434 && exec ollama run llama3"
+    # command: >
+    #   sh -c "ollama start llama3 --port 11434 && exec ollama run llama3"
+    command: ["start"]
```

Comment:
This LGTM

Reply:
Have you tried testing the API in docker env after this change?

Reply:
yes, the API works, and returns a response

Reply:
<img width="700" alt="image" src="https://github.com/Sweetdevil144/Cat-GPT/assets/40538506/e0f9316c-a5a5-42a1-bdfd-0dce81fc474d">